lib/data_update_scripts/20200826075937_migrate_profile_field_groups.rb

```diff
@@ -0,0 +1,13 @@
+module DataUpdateScripts
+  class MigrateProfileFieldGroups
+    def run
+      profile_field_groups = Hash.new do |hash, key|
+        hash[key] = ProfileFieldGroup.find_or_create_by(name: key)
+      end
+
+      ProfileField.find_each do |profile_field|
+        profile_field.update(profile_field_group: profile_field_groups[profile_field.group])
```

Comment:
Question regarding data update scripts: We will want to remove the `group` from `ProfileField` soon, which means this script will no longer work. Have we encountered a situation like this in the past? If yes, how did we deal with it. If no, is a simple guard like `return unless "group".in?(ProfileField.column_names)` at the top of `call` sufficient?

Reply:
We have dealt with it a couple of different ways. you can [add a guard clause](https://github.com/forem/forem/blob/master/lib/data_update_scripts/20200226193303_backfill_column_for_broadcasts.rb#L4) or you can [comment it out](https://github.com/forem/forem/blob/master/lib/data_update_scripts/20200326145114_re_index_feed_content_to_elasticsearch.rb#L4) or you could [even delete it](https://github.com/forem/forem/blob/master/lib/data_update_scripts/20200729120730_remove_orphaned_ahoy_events.rb). I personally like having the history so I would go for option 1 or 2.

Reply:
Thanks @mstruve! I'm almost always in favor of preserving history, so guard clause it is.

Reply:
Guard clause added.